scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -261,23 +262,45 @@ get_system_and_docker_info(){
   echo "---DEBUG INFO---"
   echo "Load: $(get_load_avg)"
   echo "CHT Containers: $(get_running_container_count)"
-  echo "Global Containers $(get_global_running_container_count)"
+  echo "Global Containers: $(get_global_running_container_count)"
+  echo "URL: $projectURL"
   echo
   echo $"$info" | column -t
 }
 
+update_nginx_local_ip_tls_cert(){
+  nginxContainerId=$1
+  rm -f /tmp/local-ip-fullchain /tmp/local-ip-key
+  curl  --retry 3 --fail --silent --show-error -o /tmp/local-ip-fullchain https://local-ip.medicmobile.org/fullchain
+  curl  --retry 3 --fail --silent --show-error -o /tmp/local-ip-key https://local-ip.medicmobile.org/key
+  docker cp /tmp/local-ip-fullchain "${nginxContainerId}":/etc/nginx/private/cert.pem  1>/dev/null
+  docker cp /tmp/local-ip-key "${nginxContainerId}":/etc/nginx/private/key.pem  1>/dev/null
+  docker exec "$nginxContainerId" bash -c "nginx -s reload"  2>/dev/null
+}
+
+validate_tls(){
+  url=$1
+  # by default curl validates TLS.  If we get back  60 then TLS isn't valid:
+  # exitcode: https://everything.curl.dev/usingcurl/verbose/writeout.html
+  # 60: https://everything.curl.dev/cmdline/exitcode.html
+  status=$(curl   --retry 3 --write-out "%{exitcode}"  -qs  "$url" -o /dev/null)
+  if [ "$status" = "60" ]; then
+    echo "false: status is $status"
+  fi
+}
```

Comment:
```suggestion
validate_tls(){
  url=$1
  attempt=$2
  max_retries=3
  # by default curl validates TLS.  If we get back  60 then TLS isn't valid:
  # exitcode: https://everything.curl.dev/usingcurl/verbose/writeout.html
  # 60: https://everything.curl.dev/cmdline/exitcode.html
  status=$(curl --retry 3 --write-out "%{exitcode}" -qs  "$url" -o /dev/null)
  if [ "$status" = "60" ]; then
    if [ $attempt -gt $max_retries ]; then
      echo "false: status is $status"
    else
      duration_seconds=$((1 * 2 ** $attempt))
      sleep $duration_seconds
      validate_tls $url $(($attempt+1))
    fi
  fi
}
```

this ad-hoc exponential backoff retry mechanism works on my machine, let me know what's your take on it